### DISCUSSION TITLE
Binary file format documentation

### DISCUSSION THREAD

**Discussion by victorkemp** [OpenRadioss/OpenRadioss#2863] [Q&A]
Hello is there any documentation for the output files like .h3d, .thy, Annn, or Tnn?

**Comment by elequiniou** [comment]
Dear Victor,

Thanks for your interest and patience. Please, have a look under:  https://github.com/OpenRadioss/Tools/tree/main/output_converters/anim_to_vtk/src and https://github.com/OpenRadioss/Tools/tree/main/output_converters/th_to_csv/src. 
This should you give advice about the format of these files by looking at their reader/converter. For H3D, format is not public (Altair proprietary format not specific to OpenRadioss).

Kind Regards,

Eric
